### PR TITLE
ReactNativeWeb BC fixed

### PR DESCRIPTION
**Pull request by tkow**
fix this issue
https://github.com/lelandrichardson/react-primitives/issues/120

Checking is available at [this repository](https://github.com/tkow/react-native-web-test).
If at the repository you change dependency of react-primitives@<0.7.0, you can see error due to using styled-components/primitive.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-primitives",
-  "version": "0.6.1",
+  "version": "0.7.0",
```
Comment (ljharb) [inline package.json]: PRs shouldn’t bump version numbers.

Comment (tkow) [reply package.json]: Sure.I'm sorry and revert it.

**Comment by msand** [comment]
Seems to work, lgtm

**Comment by geminiyellow** [comment]
good, rn4w 8.x is too old.

**Comment by kevinwolfcr** [comment]
Thanks! Any plans on releasing this soon?

**Comment by davidpett** [comment]
running into this as well. this update of rnw will fix my issues too. are there any collaborators still active in this repo to merge and release? @mathieudutour is it just you now?

**Comment by geminiyellow** [comment]
looks no good. this repo have no enough collaborator.

**Pull request merged**